### PR TITLE
updated my slide caption

### LINKED ISSUE #3: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @aloisbarreras
- [ ] @nogorilla
- [ ] @dberzano
- [ ] @normandb85630
- [ ] @sandiritter
- [ ] @LucasFrazier
- [ ] @joshuarose
- [ ] @alexxmitchell
- [ ] @osbornea
- [ ] @johndavidback
- [ ] @brookesargent
- [ ] @joshkissel
- [ ] @rainbowpeppercorn
- [ ] @mpraksti-alchemy
- [ ] @amitpng
- [ ] @brittmmendez
- [ ] @lmariel
- [ ] @waldreps
- [ ] @garnerp
- [ ] @Pawel-Prokop
- [ ] @g00g0l
- [ ] @LamarrD
- [ ] @msvaswani25
- [ ] @samwiskow
- [ ] @EduSemensati
- [ ] @gsfd2000
- [ ] @djredman99
- [ ] @binitan
- [ ] @polanczykp
- [ ] @aloisbarreras
- [ ] @mpraksti-alchemy
- [ ] @AnithaSJagadishwara
- [ ] @EduSemensati
- [ ] @AnaNunezCerda
- [ ] @rjwink

### PR DESCRIPTION
**Pull request by rainbowpeppercorn**
Octocat caption has been updated. 

Closes #3

### CODE REVIEW

File: _posts/0000-01-14-rainbowpeppercorn.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _posts/0000-01-14-rainbowpeppercorn.md]:
```suggestion
title: "Welcome to rainbowpeppercorn's slide!"
```

File: _posts/0000-01-14-rainbowpeppercorn.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 ---
 
-Caption here
+How many cats is too many cats? (There's no such number.)
```
Comment (beardofedu) [inline _posts/0000-01-14-rainbowpeppercorn.md:6]: This is a pretty solid caption! 🐱 🐈

File: _posts/0000-01-14-rainbowpeppercorn.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
 ![labtocat](https://octodex.github.com/images/labtocat.png)
```
Comment (beardofedu) [inline _posts/0000-01-14-rainbowpeppercorn.md:8]: Maybe we could use this image for the caption? ```https://octodex.github.com/images/repo.png```

image

**Review by beardofedu — CHANGES_REQUESTED** [review]

File: _posts/0000-01-14-rainbowpeppercorn.md (hunk 3 above)
Comment (rainbowpeppercorn) [reply _posts/0000-01-14-rainbowpeppercorn.md:8]: Into it.

**Review by beardofedu — APPROVED** [review]

**Pull request merged**